src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/JobStatus.java

```diff
@@ -291,7 +291,14 @@ public void addPatches(String toolName, List<RepairPatch> patches) {
         }
         return allPatches;
     }
-
+    
+    
+    protected List<RepairPatch> getCorrectbessLabeledPatches(Features features, Long buildId) {
```

Comment:
Do you mean `getCorrectnessLabeledPatches`  (i.e., getCorrect**n**essLabeledPatches) instead of getCorrect**b**essLabeledPatches?